src/js/pyodide.ts

```diff
@@ -175,7 +180,10 @@ function calculateIndexURL(): string {
   } catch (e) {
     err = e;
   }
-  const fileName = ErrorStackParser.parse(err)[0].fileName!;
+  let fileName = ErrorStackParser.parse(err)[0].fileName!;
+  if (IN_NODE && fileName.startsWith("file://")) {
+    fileName = fileName.slice("file://".length);
```

Comment:
This could use an explanatory comment about why we are doing this.

Reply:
Well we get `file://path` but node expects paths not to start with `file://`

Reply:
OK, and I assume this would also work when importing pyodide js from local files? Could you please add a comment that this is necessary when determining indexURL with remote URLs in node?

Reply:
> I assume this would also work when importing pyodide js from local files?

Well yeah. The path shouldn't ever start with `file://` in Node. If it starts with `file://` we remove it, if not we leave it alone. I think a sort of related issue is that in `compat.ts` in node we check if the path starts with `https://` in which case we make a web request otherwise we make a fs operation. But the fs operations get mad about paths starting with `file://`. Maybe a better solution would be to remove this in `compat.ts`.